panda/lib/components/text_editor/text_editor.html

```diff
@@ -0,0 +1,4 @@
+<material-spinner *ngIf="!loaded"></material-spinner>
+<div *ngIf="loaded" [id]="uniqueId" style="height: 320px; width: 100%;">
```

Comment:
It would be best if we moved the styles in a separate file

Reply:
Resolved in commit bcbe59396551c5fb9976fe12ade5168c04ae66e2.